src/app/InteractionModelEngine.cpp

```diff
@@ -462,13 +462,14 @@ Status InteractionModelEngine::OnInvokeCommandRequest(Messaging::ExchangeContext
 }
 
 CHIP_ERROR InteractionModelEngine::ParseAttributePaths(const Access::SubjectDescriptor & aSubjectDescriptor,
-                                                       AttributePathIBs::Parser & aAttributePathListParser,
+                                                       AttributePathIBs::Parser & aAttributePathListParser, bool & aPathsIsNotEmpty,
```

Comment:
Blocker comment to allow time for a careful review on logic on this PR as the underlying assumptions are reasonably deep.

Overall I wonder if we should have the "deny on no ACL" at all throughout ...